### PR TITLE
Bumped Love2d to version 11.3

### PR DESCRIPTION
**Pull request by Cronos87**

### CODE REVIEW

File: docsets/Love2d/docset.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "Love2d",
-    "version": "11.2",
+    "version": "11.3",
     "archive": "love2d.tgz",
```
Comment (Kapeli) [inline docsets/Love2d/docset.json]: You need to update the archive at this path as well.

Comment (Cronos87) [reply docsets/Love2d/docset.json]: Thanks. I just change it.

**Comment by Kapeli** [comment]
Thank you! 👍

**Comment by Cronos87** [comment]
Thank you too!

**Pull request merged**